src/swap/types.ts

```diff
@@ -93,9 +93,13 @@ export type LifecycleStatus =
   | {
       statusName: 'amountChange';
       statusData: {
-        amountFrom: string;
+        amountFrom?: string;
+        amountETH?: string;
+        amountUSDC?: string;
         amountTo: string;
         tokenFrom?: Token;
+        tokenFromETH?: Token;
+        tokenFromUSDC?: Token;
```

Comment:
nit: since swap lifecycle now supports two usecases, might be useful to have comments here explaining whats valid for each